### PR TITLE
python3Packages.coinmetrics-api-client: 2023.2.23.0 -> 2023.5.2.20

### PR DESCRIPTION
**Pull request by centromere**
###### Description of changes

[Commits](https://github.com/coinmetrics/api-client-python/commits/master).

Also:
* `nix-update-script` was enabled
* Type checking is now part of the tests

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mweinelt** [comment]
Please don't request my review for cryptocurrency packages.

File: pkgs/development/python-modules/coinmetrics-api-client/default.nix [hunk 1]
```diff
@@ -38,18 +46,27 @@ buildPythonPackage rec {
   ];
 
   nativeCheckInputs = [
+    mypy
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/coinmetrics-api-client/default.nix:49]: We don't really want to run linting tools in checkPhase

Comment (centromere) [reply pkgs/development/python-modules/coinmetrics-api-client/default.nix:49]: Which phase do you believe is most appropriate?

Comment (centromere) [reply pkgs/development/python-modules/coinmetrics-api-client/default.nix]: After consulting with @mweinelt, I've removed `mypy` entirely.

Comment (mweinelt) [reply pkgs/development/python-modules/coinmetrics-api-client/default.nix]: In unsubscribed and you resubscribed me :cry:

**Comment by SuperSandro2000** [comment]
@ofborg build python310Packages.coinmetrics-api-client

**Pull request merged**